modules/machine-config-daemon-metrics.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-machine-config-daemon-metrics.adoc
+
+[id="machine-config-daemon-metrics_{context}"]
+= Machine Config Daemon metrics
+
+Beginning {product-title} 4.3, the Machine Config Daemon provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.
+
+The following table describes this set of metrics. Note that:
+
+* Metrics marked with `*` in the `Name` and `Description` columns represent serious errors that might cause performance problems. Such problems might prevent updates and upgrades from proceeding.
+* While some entries contain commands for getting specific logs, the most comprehensive set of logs is available using the `oc adm must-gather` command.
+
+[cols="1,1,2,2", options="header"]
+.MCO metrics
+|===
+|Name
+|Format
+|Description
+|Notes
+
+|mcd_host_os_and_version
+|[]string{"os", "version"}
+|Shows the OS that MCD is running on, such as RHCOS or RHEL. In case of RHCOS, the version is provided.
+|
+
+|ssh_accessed
+|counter
+|Shows the number of successful SSH authentications into the node.
+|The non-zero value shows that someone might have made manual changes to the node. Such changes might cause irreconcilable errors due to the differences between the state on the disk and the state defined in the machine configuration.
+
+|mcd_drain*
+|{"drain_time", "err"}
+|Logs errors received during failed drain. *
+|While drains might need multiple tries to succeed, terminal failed drains prevent updates from proceeding. The `drain_time` metric, which shows how much time the drain took, might help with troubleshooting. For further investigation, see the logs by running:
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+|mcd_pivot_err*
+|[]string{"pivot_target", "err"}
+|Logs errors encountered during pivot. *
+|Pivot errors might prevent OS upgrades from proceeding. For further investigation, run these commands to access the node and see its logs:
+
+`$ oc debug node <node>`
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+`$ journalctl -u pivot.service`
+
+|mcd_state
+|[]string{"state", "reason"}
+|State of Machine Config Daemon for the indicated node. Possible states are "Done", "Working", and "Degraded". In case of "Degraded", the reason is included.
+|For further investigation, see the logs by running:
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+|mcd_kubelet_state*
+|[]string{"err"}
+|Logs kubelet health failures.  *
+|This is expected to be empty, with failure count of 0. If failure count exceeds 2, the error indicating threshold is exceeded. This indicates a possible issue with the health of the kubelet. For further investigation, run these commands to access the node and see its logs:
+
+`$ oc debug node <node>`
```

Comment:
This is in the correct order, but we could make it more explicit with the addition of the `chroot /host` command as noted above.